packages/protocol/scripts/foundry/deploy_libraries.sh

```diff
@@ -54,6 +54,7 @@ for LIB_PATH in "${LIBRARIES_PATH[@]}"; do
     # LIB_NAME = AddressSortedLinkedListWithMedian
     echo "Deploying library: $LIB_NAME"
     create_library_out=`forge create $LIB_PATH --from $FROM_ACCOUNT --rpc-url $ANVIL_RPC_URL --unlocked --broadcast --json`
+    echo "Create library out: $create_library_out" 
```

Comment:
For consistency I would keep it uppercase
```suggestion
CREATE_LIBRARY_OUT=`forge create $LIB_PATH --from $FROM_ACCOUNT --rpc-url $ANVIL_RPC_URL --unlocked --broadcast --json`
    echo "Create library out: $CREATE_LIBRARY_OUT" 
```